runtime/doc/vim_diff.txt

```diff
@@ -265,6 +273,8 @@ Lua interface (|if_lua.txt|):
 on cancel and completion respectively) via dictionary argument (replaces all 
 other arguments if used).
 
+|input()| and |inputdialog()| now support user-defined cmdline highlighting.
```

Comment:
Though no, arabic characters fall under different category: “hard to implement, unlikely to happen and should really be solved on the different layer and not with some hacks in ex_getln.c and screen.c (it contains another incarnation of arabic shaping code)”. What falls into “causes problems” category now are hooks which throw errors.